docs/cluster-configuration.md

```diff
@@ -1524,3 +1524,19 @@ spec:
   instancetypeConfig:
     referencePolicy: expand
 ```
+## KubeVirt Common Instance Types Deployment Configuration
+
+The configuration of [common instance types deployment](https://kubevirt.io/user-guide/user_workloads/deploy_common_instancetypes/) is held separately to the core instance type and preference functionality within the [KubeVirt CR](https://kubevirt.io/api-reference/main/definitions.html#_v1_commoninstancetypesdeployment). This is mirrored in the HCO CR through the `spec.commonInstancetypesDeployment` configurable.
```

Comment:
@lyarwood - maybe we need to also say what's is the default behavior? IIUC, it's like setting this new field to `true`?